packages/panzer/dof-mgr/test/cartesian_topology/CartesianConnManager_impl.hpp

```diff
@@ -310,7 +310,7 @@ getBlockId(LocalOrdinal localElmtId) const
 }
 
 template <typename LocalOrdinal,typename GlobalOrdinal>
-CartesianConnManager<LocalOrdinal,GlobalOrdinal>::Triplet<int> 
+typename CartesianConnManager<LocalOrdinal,GlobalOrdinal>::template Triplet<int> 
```

Comment:
I suppose I'm fine with this, but would you mind explaining the change?  I've never quite understood why some things need to be `something::template somethingElse` as opposed to just `something::somethingElse`.

Reply:
The "template" keyword is used to tell the compiler that the next angle bracket is a template, not a less than sign. It's needed because the Triplet object could change meaning depending on the template parameters. This code has been around for 2 years, compiling on a bunch of different platforms with different compilers and we just saw our first issue with this. I guess it's just clang being picky about enforcing the standard?

Reply:
Ah, gothca.